### PR TITLE
Add symbol load status to module load event message

### PR DESCRIPTION
**Pull request by wesrupert**

### CODE REVIEW

**Comment by wesrupert** [comment]
@caslan @chuckries @rajkumar42

**Comment by chuckries** [comment]
:+1:

File: src/MIDebugEngine/ResourceStrings.resx [hunk 1]
```diff
@@ -147,6 +147,24 @@
...
+    <value>Unloaded '{0}'.</value>
+  </data>
+  <data name="ModuleLoadedWithSymbols" xml:space="preserve">
+    <value>Symbols loaded.</value>
+  </data>
+  <data name="SymbolsLoadedInfo" xml:space="preserve">
+    <value>Symbols loaded - {0}</value>
+  </data>
+  <data name="ModuleLoadedWithoutSymbols" xml:space="preserve">
+    <value>Cannot find or open the PDB file.</value>
```
Comment (caslan) [inline src/MIDebugEngine/ResourceStrings.resx]: Will this message show up for C++ extension? If so, you probably don't want to use 'PDB'. Maybe something like 'Cannot find or open the symbol file.'

**Comment by caslan** [comment]
LGTM except the resource string.

**Pull request merged**